packages/firestore/src/remote/remote_store.ts

```diff
@@ -275,12 +295,23 @@ export class RemoteStore implements TargetMetadataProvider {
    */
   private shouldStartWatchStream(): boolean {
     return (
-      this.isNetworkEnabled() &&
-      !this.watchStream.isStarted() &&
+      this.shouldStartWatchStreamAfterError() &&
       !objUtils.isEmpty(this.listenTargets)
     );
   }
 
+  /**
+   * Returns whether the watch stream should be started after an error because
+   * it's necessary and has not yet been started. Similar to
+   * {@link shouldStartWatchStream} except this one does not check whether there
```

Comment:
I'm actually not sure. There is other `{@link }` annotation in the `.ts` code already so I assume it works?

Reply:
FWIW- Since we don't actually generate docs from our comments it doesn't matter, but `@link` is valid in JSDoc (http://usejsdoc.org/tags-inline-link.html).